Typing error, string[] cannot be assigned to PrezFlavour[]

Building a prod release:
```sh
npm run build
```

Error:
```sh
src/main.ts:17:36 - error TS2345: Argument of type 'string[]' is not assignable to parameter of type 'PrezFlavour[]'.
  Type 'string' is not assignable to type 'PrezFlavour'.

17 app.provide(enabledPrezsConfigKey, config.enabledPrezs.split(","));
```
This is the referenced line:
https://github.com/RDFLib/prez-ui/blob/781622441d0c2ecd60200088c0352e82c547cc61/src/main.ts#L17

My build environment is using:
`vite` v4.5.3
`typescript` v5.2.2

--- a/src/main.ts
+++ b/src/main.ts
@@ -4,6 +4,7 @@ import App from "@/App.vue";
 import router from "@/router";
 import config from "@/config";
 import { sidenavConfigKey, enabledPrezsConfigKey, apiBaseUrlConfigKey, mapConfigKey, perPageConfigKey, conceptPerPageConfigKey, enableScoresKey } from "@/types";
+import type { PrezFlavour } from "@/types";
 import { Tooltip } from "floating-vue";
 import VueGoogleMaps from "@fawmi/vue-google-maps";
 
@@ -14,7 +15,7 @@ import "@/assets/sass/main.scss";
 const app = createApp(App);
 
 app.provide(sidenavConfigKey, config.sidenav === "true");
-app.provide(enabledPrezsConfigKey, config.enabledPrezs.split(","));
+app.provide(enabledPrezsConfigKey, (config.enabledPrezs.split(",") as PrezFlavour[]));
 app.provide(perPageConfigKey, config.perPage);
 app.provide(conceptPerPageConfigKey, config.conceptPerPage);
 app.provide(enableScoresKey, config.enableScores === "true");

